tests/tests/test-stake.ts

```diff
@@ -19,6 +21,28 @@ describeWithMoonbeam("Moonbeam RPC (Stake)", `simple-specs.json`, (context) => {
 
   it("candidates set in genesis", async function () {
     const candidates = await context.polkadotApi.query.stake.candidates(GENESIS_ACCOUNT);
-    expect((candidates.toHuman() as any).id.toLowerCase()).equal(GENESIS_ACCOUNT);
+    expect(candidates.toHuman()["id"].toLowerCase()).equal(GENESIS_ACCOUNT);
+    expect(candidates.toHuman()["state"]).equal("Active");
+  });
+
+  it("inflation set in genesis", async function () {
+    const inflationInfo = await context.polkadotApi.query.stake.inflationConfig();
+    // {
+    //   expect: {
+    //     min: '100.0000 kUnit',
+    //     ideal: '200.0000 kUnit',
+    //     max: '500.0000 kUnit'
+    //   },
+    //   round: { min: '0.00%', ideal: '0.00%', max: '0.00%' }
+    // }
+    expect(inflationInfo.toHuman()["expect"]["min"]).to.eq("100.0000 kUnit");
+    expect(inflationInfo.toHuman()["expect"]["ideal"]).to.eq("200.0000 kUnit");
+    expect(inflationInfo.toHuman()["expect"]["max"]).to.eq("500.0000 kUnit");
+    expect(inflationInfo.toHuman()["round"]["min"]).to.eq("0.00%");
+    expect(Number(inflationInfo["round"]["min"])).to.eq(4563); // 4% / 8766 * 10^9
+    expect(inflationInfo.toHuman()["round"]["ideal"]).to.eq("0.00%");
+    expect(Number(inflationInfo["round"]["ideal"])).to.eq(5703); // 5% / 8766 * 10^9
+    expect(inflationInfo.toHuman()["round"]["max"]).to.eq("0.00%");
+    expect(Number(inflationInfo["round"]["max"])).to.eq(5703); // 5% / 8766 * 10^9
```

Comment:
@4meta5 I'm curious: what is the 8766 corresponding to?

Reply:
The number of hours in a year. Each round is 1 hour so this is the number of rounds in a year.

Reply:
thanks ^^